core/modules/simpletest/tests/batch_test.module

```diff
@@ -357,7 +357,7 @@ function _batch_test_batch_0() {
 function _batch_test_batch_1() {
   // Ensure the batch takes at least two iterations.
   $total = 10;
-  $sleep = (1000000 / $total) * 2;
+  $sleep = 100000 / $total;
```

Comment:
I believe that these were set to 2sec for good reasons. We may have failures in slower systems (not 100% though - someone else please verify?)